### ISSUE TITLE
cordova build failed after adding this plugin

### ISSUE THREAD

**Issue by krestia** [EMI-INDO/emi-indo-cordova-plugin-admob#9]
i try to build with hello cordova and success build an .apk, but when i add this plugin, it always fail

BUILD FAILED in 18s
D:\5.GAME\Cordova\1new\hello\platforms\android\gradlew: Command failed with
exit code 1 Error output:
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugResources'.
> Android resource linking failed
  D:\5.GAME\Cordova\1new\hello\platforms\android\app\build\intermediates\mer
ged_manifests\debug\AndroidManifest.xml:23: AAPT: error: unexpected element <que
ries> found in <manifest>.

**Comment by krestia** [comment]
@EMI-INDO 

D:\5.GAME\Cordova\1new\hello>cordova --version
10.0.0 (cordova-lib@10.1.0)

D:\5.GAME\Cordova\1new\hello>cordova platform is
Installed platforms:
  android 8.1.0
Available platforms:
  browser ^6.0.0
  electron ^1.1.1
  windows ^7.0.0

D:\5.GAME\Cordova\1new\hello>cordova plugin is
cordova-plugin-whitelist 1.3.5 "Whitelist"
emi-indo-cordova-plugin-admob 1.0.9 "emiAdmobPlugin"

i also used androidx and androidx adapter plugin, but doesn't solved the issue

**Comment by EMI-INDO** [comment]
@krestia 

## remove
- cordova platform rm android
- cordova plugin rm emi-indo-cordova-plugin-admob

## add new
- cordova platform add android@11.0.0
- cordova plugin add emi-indo-cordova-plugin-admob


If there are still problems contact me back

**Comment by krestia** [comment]
@EMI-INDO 

Hi, many thanks

it is work!

D:\5.GAME\Cordova\1new\hello>cordova --version
10.0.0 (cordova-lib@10.1.0)

D:\5.GAME\Cordova\1new\hello>cordova platforms is
Installed platforms:
  android 11.0.0
Available platforms:
  browser ^6.0.0
  electron ^1.1.1
  windows ^7.0.0

D:\5.GAME\Cordova\1new\hello>cordova plugin is
cordova-plugin-whitelist 1.3.5 "Whitelist"
emi-indo-cordova-plugin-admob 1.0.9 "emiAdmobPlugin"

**Comment by EMI-INDO** [comment]
@krestia 

- did you build the apk successfully?

If you are using cordova 11.0.0  please delete cordova-plugin-whitelist
- `cordova plugin rm cordova-plugin-whitelist`

- add in config.xml 
- AndroidXEnabled = true
```
<widget>
    <platform name="android">
        <preference name="AndroidXEnabled" value="true" />
    </platform>
</widget>
```

## If you fail to build apk
 - try
 upgrade JVM: 8 to JVM: 11

- please always provide error log information

**Comment by krestia** [comment]
@EMI-INDO

yes i build an .apk succesfully with your suggestion.

i used your template code (.js) and (.html) but the ads not showing. 
because i am not sure what i.m missing, show i don't mention it in my reply before.

**your sugestion** : If you are using cordova 11.0.0 please delete cordova-plugin-whitelist

cordova plugin rm cordova-plugin-whitelist

add in config.xml

AndroidXEnabled = true

<widget>
    <platform name="android">
        <preference name="AndroidXEnabled" value="true" />
    </platform>
</widget>

ok, i'll try it, may it is cause that error

**Comment by krestia** [comment]
@EMI-INDO 

thanks for your information, but now i'm really confusing about the resut:

1. i don't make any change in my config.xml as before
**RESULT : 
a) Intertitial, and banner not showing
b) showing alert :** 
![alert](https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/assets/37997005/56d02380-5b5f-4f36-93dc-043fd6d3f5f6)


2. I read the new issue : [https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/issues/10]
    firts : i modified the index.html , insert button :

```
<p> <button onclick="loadInterstitialAd();">Load Interstitial Ad</button></p>
<p> <button onclick="showInterstitialAd();">Show Interstitial Ad</button></p>
<p> <button onclick="showBannerAdaptive();">show adaptive</button></p>
<p> <button onclick="showBannerNotAdaptive();">Show banner</button></p>
```

4. Inserting the code from : [https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/blob/main/example/auto_load_and_show.html](url)   in my index.js

```
// For example automatically load Interstitial Ad

// Load Interstitial Ad

let loadInterstitialAd = () => {
    cordova.plugins.emiAdmobPlugin.loadInterstitialAd(
    AdUnitId = "ca-app-pub-3940256099942544/1033173712",
	npa = "1", // String | 0 | 1
    responseInfo = true, // boolean

    // Optional
	(info) => { alert(info) }, 
    (error) => { alert(error)
    
    });

    // call loadInterstitialAd();
} //batas blok dari let loadInterstitialAd

// Show Interstitial Ad = variable for showing intertitial ads

let showInterstitialAd = () => {
    cordova.plugins.emiAdmobPlugin.showInterstitialAd();

    // call showInterstitialAd();
}
```


5. Adding the banner variable in index.js 

```
//  Banner Adaptive

let showBannerAdaptive = () => {
    cordova.plugins.emiAdmobPlugin.showBannerAd(
    AdUnitId = "ca-app-pub-3940256099942544/6300978111",
    npa = "1", // String | 0 | 1
    position = "bottom-center",
    size = "Anchored_FULL_WIDTH", // | Inline_adaptive | Anchored_adaptive
    adaptiveWidth = 320,
    responseInfo = true, // boolean (debugging)
    
    // Optional
    (info) => { alert(info) },
    (error) => { alert(error)

    });
}

// call showBannerAdaptive();

//  not Adaptive banner

let showBannerNotAdaptive = () => {
    cordova.plugins.emiAdmobPlugin.showBannerAd(
    AdUnitId = "ca-app-pub-3940256099942544/6300978111",
    npa = "1", // String | 0 | 1
    position = "bottom-center",
    size = "FLUID",
    responseInfo = true, // boolean (debugging)
    
    // Optional
    (info) => { alert(info) },
    (error) => { alert(error)

    });
}

// call showBannerNotAdaptive();`


`document.addEventListener("deviceready", function(){

// Before loading ads, have your app initialize the Google Mobile Ads SDK by calling
// This needs to be done only once, ideally at app launch.

cordova.plugins.emiAdmobPlugin.initialize();


// auto load call loadInterstitialAd(); 

 loadInterstitialAd(); //memanggil let loadInterstitialAd()

//after call event :
// Interstitial EVENT

/*
on.InterstitialAdLoaded
on.InterstitialAdFailedToLoad
on.InterstitialAdClicked
on.InterstitialAdDismissedFullScreenContent
on.InterstitialAdFailedToShowFullScreenContent
on.InterstitialAdImpression
on.InterstitialAdShowedFullScreenContent
*/

document.addEventListener('on.InterstitialAdLoaded', () => {

    // auto show call showInterstitialAd();

    showInterstitialAd();
    //showBannerNotAdaptive();
    showBannerAdaptive();

    console.log("on Interstitial Ad Loaded");

});




}, false); //end of event deviceready
```

**RESULT : 
1. Alert : 
![alert2](https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/assets/37997005/fcc426b8-5aa6-4c4b-a223-4a39f4fe80ff)

2. interstitial not showing
3. banner show after click the button (not auto)
4. banner adaptive did't showing, even i click the button**

![banner](https://github.com/EMI-INDO/emi-indo-cordova-plugin-admob/assets/37997005/6a9b8c46-7c7f-4c26-9cb6-b28a9c34cc33)


**what i'm missing ?**
 the interstitial not show
the banner adaptive not show
banner show after click the button (not auto)

**Comment by krestia** [comment]
@EMI-INDO 
Thanks, I take down my comment before.
 
with last setting, my interstitial ads showed but in blank screen. 
For me, it because of the ads (function) do not work in eventlistener function (document.addEventListener("deviceready", function()) through index.html, but it must called in runtime.

your plugin and example working good, it is only in my app that need some adjustment. 

i can close this issue
